src/assets/mapStyles/waterTemperature/mapStyles.js

```diff
@@ -183,8 +104,8 @@ export default {
                     "line-color": [
                         "interpolate", ["linear"], ["get", "temp"],
                         0, "#10305d", 
-                        14.08, "#c4c1b6",
-                        25.88, "#730000"
+                        15, "#c4c1b6",
```

Comment:
this is in prep for the legend edit task, but figured i would tweak it while I'm in here.